src/status_im/contexts/wallet/collectible/events.cljs

```diff
@@ -185,7 +185,7 @@
            ;; We delay the navigation because we need re-frame to update the DB on time.
            ;; By doing it, we skip a blink while visiting the collectible detail page.
            [:dispatch-later
-            {:ms       1
+            {:ms       17
```

Comment:
on iOS 1 ms wasn't enough, this has been increased to avoid the blinking. This is unnoticeable for the user.

Reply:
Is `17ms` always going to be enough?

Reply:
Hey Omar!
This is just slightly more than a frame (running at 60fps)
Ofc I could set 50ms, but right now, as far as I tested, this works well.